nixos/modules/services/monitoring/uptime-kuma.nix

```diff
@@ -0,0 +1,108 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.services.uptime-kuma;
+in
+{
+
+  options = {
+    services.uptime-kuma = {
+      enable = mkEnableOption (mdDoc "Uptime Kuma");
```

Comment:
Presumably the intention is to run this behind a reverse proxy. Can you please explain that here so it is clear to potential users.